### PR TITLE
Logging bug fix and optimisation

### PR DESCRIPTION
**Pull request by KB-perByte**
##### SUMMARY

Fixes wrong ordering of commands and ipv6 host command rendering

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

ios_logging.py

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by KB-perByte** [comment]
recheck

**Comment by KB-perByte** [comment]
recheck

**Comment by KB-perByte** [comment]
recheck

File: plugins/module_utils/network/ios/config/logging_global/logging_global.py [hunk 1]
```diff
@@ -137,85 +111,66 @@ def _compare(self, want, have):
...
+        for el in list_el:
+            if tmp_data.get(el):
+                tmp_data[el] = {
+                    self.trim_whitespace(el + i): {el: i} for i in tmp_data[el]
+                }
+        return tmp_data
 
     def trim_whitespace(self, word):
-        return word.strip()
+        return str(word).strip()
```
Comment (NilashishC) [inline plugins/module_utils/network/ios/config/logging_global/logging_global.py:176]: Should we use to_text() here instead?

**Review by NilashishC — APPROVED** [review]

**Comment by KB-perByte** [comment]
recheck

**Comment by KB-perByte** [comment]
recheck

**Comment by KB-perByte** [comment]
recheck

**Comment by KB-perByte** [comment]
recheck

**Comment by KB-perByte** [comment]
recheck

**Pull request merged**